packages/mwp-app-render/src/components/connectedWithMatchMedia.jsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { createStructuredSelector, createSelector } from 'reselect';
+import withMatchMedia from 'meetup-web-components/lib/utils/components/withMatchMedia';
+
+/**
+ * getConfig
+ * @param  {Object} state
+ * @return {Object} the config from state
+ */
+const getConfig = state => state.config || {};
+
+/**
+ * Checks if object is empty
+ * @param  {Object} object
+ * @return {Boolean} whether object is empty
+ */
+const isEmpty = object => object === undefined || Object.keys(object).length === 0;
+
+/**
+ * @param {Object} device - object containing device information
+ * @returns {Object} - default media object for `withMatchMedia`
+ */
+const getMediaOrDefault = (media, device) => {
+	if (!isEmpty(media)) {
+		return media;
+	}
+	if ( !isEmpty(device) ) {
```

Comment:
MWP has total control over the existence of this value - we can assume `state.config.device` is never empty and can enforce it with Flow types